src/Symfony/Component/DependencyInjection/Compiler/AutowirePass.php

```diff
@@ -540,15 +540,19 @@ private function createTypeNotFoundMessage(TypedReference $reference, string $la
         if (!$r = $this->container->getReflectionClass($type, false)) {
             // either $type does not exist or a parent class does not exist
             try {
-                $resource = new ClassExistenceResource($type, false);
-                // isFresh() will explode ONLY if a parent class/trait does not exist
-                $resource->isFresh(0);
-                $parentMsg = false;
+                if (class_exists(ClassExistenceResource::class)) {
+                    $resource = new ClassExistenceResource($type, false);
+                    // isFresh() will explode ONLY if a parent class/trait does not exist
+                    $resource->isFresh(0);
+                    $parentMsg = false;
+                } else {
+                    $parentMsg = "couldn't be loaded, installing symfony/config package could help with additional details";
```

Comment:
`couldn't be loaded. Either it was not found or it is missing a parent class or a trait.`?

Reply:
I like this, updated!